contracts/campaignAirdrops/ExistingERC20DropCampaign.sol

```diff
@@ -58,18 +62,27 @@ contract ExistingERC20DropCampaign is CampaignAidropsShared {
         emit AirdropFunded(address(this));
     }
 
+    /// @notice Allows admin to enable the funder address to withdraw funds early if no addresses are eligible to claim
+    function unlockWithdraw() external onlyAdmin {
```

Comment:
Maybe instead of this method, you can decide to disable funding if the merkleRoot has not been set

Reply:
Well the situation is: Funding is always supposed to happen at the beginning, when the creator is creating the campaign. When the campaign ends the backend sends the merkleRootHash and then, that is when the 60 day claim period starts, period where claimers can claim their rewards - but, during this claim period the funder cannot withdraw their funds ->. this unlockWithdraw method is for the situation when either in the middle or in the end of the campaign there are no eligible accounts to claim rewards, and the merkleRootHash won't even be sent -> so we need a way to allow the funder to withdraw their funds and a way for the admin to unlock this for him, without sending a merkleRootHash @bxmmm1

Reply:
oh, ok then